### DISCUSSION TITLE
Model name wave file

### DISCUSSION THREAD

**Discussion by tonyyng** [EdgeTX/edgetx#2169] [Q&A]
With my Jumper T16 on OpenTx, it use to play /SOUNDS/en/my_model/name.wav when I loaded "my model".  This doesn't seem to do the same thing on EdgeTx.

Is there something I need to configure or was this something that wasn't carried forward?  Anyone have this working?

**Comment by pfeerick** [comment]
I'm just having a quick look at this now in the simulator, and will look properly on the radio tomorrow. But does it currently play the model name of the selected model when powering on the radio, but no play it when changing models... as this seems the be the behaviour I'm seeing on the simulator. If so, this should be simple to fix.

**Comment by tonyyng** [comment]
I'm not getting the name.wav playing in any situation:  simulator or radio, starting the simulator/radio or switching models on the simulator or radio.

**Reply by pfeerick** [reply]
With a name of "My Model", try "\SOUNDS\en\My Model\name.wav" - it looks like it correctly handles spaces now, and doesn't need the underscore. But still, only for the default model at power on.

**Comment by tonyyng** [comment]
I removed the underscore and the sound plays at power on.  Perhaps you should continue to support the underscore, since all the radios coming from OpenTX will have it setup with underscores.  Or document the correct setup somewhere if you don't want to carry forward the underscore.

Thanks for your help

**Reply by pfeerick** [reply]
See https://github.com/EdgeTX/edgetx/pull/2196 ... this was a side effect of another PR from a year ago to do with model notes which also touch on the audio file paths... I intend to make it so both forms are accepted. Also, fix up the fact it isn't playing when changing models atm ;)
